docs/dev-setup/devnet-setup.md

```diff
@@ -1,31 +1,70 @@
-## Setting Up a Network For Development
+## Setting Up a Network
 
-This document covers setting up a network on your local machine for development using Docker containers.
+This document covers setting up a network on your local machine for various development and testing activities. Unless you are intending to contribute to the development of the Hyperledger Fabric project, you'll probably want to follow the more commonly used approach below - [leveraging published Docker images](#leveraging-published Docker-images) for the various Hyperledger Fabric components, directly. Otherwise, skip down to the [secondary approach](#building-your-own-images) below.
 
-All commands should be run from within the Vagrant environment described in [Setting Up Development Environment](devenv.md).
-See [Logging Control](logging-control.md) for information on controlling
-logging output from the `peer` and chaincodes.
+### Leveraging published Docker images
 
+This approach simply leverages the Docker images that the Hyperledger Fabric project publishes to [DockerHub](https://hub.docker.com/u/hyperledger/dashboard/) and either Docker commands or Docker Compose descriptions of the network one wishes to create.
```

Comment:
Link is 404 for me. https://hub.docker.com/u/hyperledger works. Maybe because I'm not logged in?